packages/dw-connector/.rubocop.yml

```diff
@@ -0,0 +1,6 @@
+inherit_gem:
+  rubocop-powerhome: rubocop.yml
+
+AllCops:
+  TargetRubyVersion: 2.7
```

Comment:
```suggestion
  TargetRubyVersion: 3.3
```

Reply:
rubocop isn't executable with the current definition of this file. What I've pasted above makes it executable. And also finds 70 offenses in the repo as it stands (68 auto-correctable).